.github/workflows/codeql.yml

```diff
@@ -1,11 +1,10 @@
 name: "CodeQL"
 
 on:
-  push:
-    branches: [ "dev" ]
-  workflow_dispatch:
-  schedule:
-    - cron: '15 14 * * 1'
+  workflow_run:
+    workflows: ["Build and test"]
+    types:
+      - completed
```

Comment:
Why is this change?

Reply:
Codeql should be trig when the first workflow is terminated, we need all the report to be upload.

Reply:
How, I think I mixed between codecov and codeql